composer.json

```diff
@@ -1,14 +1,13 @@
 {
-    "name": "nics/nidirect",
```

Comment:
Do we want to keep any custom metadata like this for the project? Probably not terribly important right now but it's not clear whether this is tied at all to any specific composer behaviour or whether it's purely information and can be changed at any time.

Reply:
Left this in as a quick reference back to the origin of the composer file and that we are no longer using the old drupal-project github project.